get_data/upload_to_db.py

```diff
@@ -77,36 +105,46 @@ def upload_to_db(label_file, bucket_name, key_prefix, es_host_ip, es_port, es_in
       {
         "img_id": "xxx",
         "file_name": "file-name.jpg",
-        "location": "/path/to/picture/dir/"
+        "location": "/path/to/picture/dir/",
+        "event": "event_name"
       }
     ]
     then you can add any field you wish to labelized the picture.
-    :param label_file:          [string]    path to the file containing the labels in json format
-    :param bucket_name:         [string]    Name of the s3 bucket
-    :param key_prefix:          [string]    Prefix for every uploaded picture to s3. Unix path system (use slashes: '/')
-                                            For example if:
-                                            bucket_name = "my-bucket", key_prefix = "subfolder/"
-                                            Every picture will be uploaded to "my-bucket/subfolder/{img-id}"
-    :param es_host_ip:          [string]    Public ip of the Elasticsearch host server
-    :param es_port:             [int]       Port open for Elasticsearch on host server (typically 9200)
-    :param es_index:            [string]    Name of the index to use
-    :param overwrite:           [bool]      If True will overwrite picture is s3 and label in ES if same img_id is found
-                                            If False (default), only non existing img_id picture will be uploaded
-    :return:                    [tuple]     (int) Number of successful upload, (int) number of failed upload
-                                            Return None on error
+    Note: if picture already exist in s3 and is not overwritten, upload to ES will be tried anyway.
+    :param label_file:      [string]    path to the file containing the labels in json format
+    :param bucket_name:     [string]    Name of the s3 bucket
+    :param es_host_ip:      [string]    Public ip of the Elasticsearch host server
+    :param es_port:         [int]       Port open for Elasticsearch on host server (typically 9200)
+    :param es_index:        [string]    Name of the index to use
+    :param overwrite:       [bool]      If True will overwrite picture is s3 and label in ES if same img_id is found
```

Comment:
je ne comprends pas la phrase

Reply:
j'ai reformulé:
If True, new picture/label will overwrite existing ones in s3/ES with same img_id

dis moi si c'est plus clair ?